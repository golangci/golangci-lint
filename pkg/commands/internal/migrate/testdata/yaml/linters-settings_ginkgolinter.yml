issues:
  # Only to not generate unrelated elements inside golden.
  exclude-use-default: false
  # Only to not generate unrelated elements inside golden.
  exclude-generated: strict
  # Only to not generate unrelated elements inside golden.
  exclude-dirs-use-default: false

linters-settings:
  ginkgolinter:
    suppress-len-assertion: true
    suppress-nil-assertion: true
    suppress-err-assertion: true
    suppress-compare-assertion: true
    suppress-async-assertion: true
    suppress-type-compare-assertion: true
    forbid-focus-container: true
    allow-havelen-zero: true
    force-expect-to: true
    validate-async-intervals: true
    forbid-spec-pollution: true
    force-succeed: true
