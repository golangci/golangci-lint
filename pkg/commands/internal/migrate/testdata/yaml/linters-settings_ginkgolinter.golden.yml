version: "2"
linters:
  settings:
    ginkgolinter:
      suppress-len-assertion: true
      suppress-nil-assertion: true
      suppress-err-assertion: true
      suppress-compare-assertion: true
      suppress-async-assertion: true
      suppress-type-compare-assertion: true
      forbid-focus-container: true
      allow-havelen-zero: true
      force-expect-to: true
      validate-async-intervals: true
      forbid-spec-pollution: true
      force-succeed: true
